src/main/java/com/ldtteam/structurize/management/Structures.java

```diff
@@ -119,7 +129,7 @@ private Structures()
      */
     public static void init()
     {
-        loadStyleMaps();
+        executor.submit(() -> loadStyleMaps());
```

Comment:
Add boolean and make it synchronous by default (because of reload method below).